buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -3,4 +3,6 @@ object Versions {
     const val compileSdk = 29
     const val minSdk = 21
     const val sqldelight = "1.4.0"
+    const val buildKonfig = "0.5.1"
+    const val mapbox = "9.2.0"
```

Comment:
Since we only have one library for each of these versions, we can just inline them (like the Android Gradle Plugin).